hw1/01-selectors.md

```diff
@@ -2,6 +2,33 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple
+5. #fancy pickle
+6. .small
+7. orange.small
+8. bento orange.small
+9. plate,bento
+10. \*
+11. plate \*
+12. plate + .small, plate + apple
+13. pickle ~ pickle
```

Comment:
So... this selects the sibling `pickle` of `pickle`, which I think select all pickles except the first one haha


what do you think about `bento ~ pickle` instead❓

Reply:
Thanks